back/engines/free/email_campaigns/app/mailers/email_campaigns/comment_on_idea_you_follow_mailer.rb

```diff
@@ -2,19 +2,64 @@
 
 module EmailCampaigns
   class CommentOnIdeaYouFollowMailer < ApplicationMailer
+    # TODO: Test that variables match the variables in the default text
+    # TODO: What about the difference based on idea term here?
+    # Maybe change the actual translations to include as a variable 'an idea' or 'a proposal'?
+    def self.campaign_class
+      Campaigns::CommentOnIdeaYouFollow
+    end
+
+    # Returns the editable regions for this campaign.
+    # These regions are used to define the custom text that can be edited by the user.
+    def self.editable_regions
```

Comment:
I really like the readability of this 👍 I wonder if it could make sense to define the substitution variables just once for the whole campaign and let the moderator decide which ones they want to use in which region? Perhaps something like:
```
def substitution_variables
  {
    input_title: resolve_multiloc(input.title_multiloc),
    author_name: author.full_name,
    ...
  }
end
```

Do you think there could be more than one copy/settings per region at some point? For example a CTA text and a CTA URL for the CTA button? In that case we might need to pass an additional copy/settings argument to `editable_region`?